frontend/src/utils/utils.ts

```diff
@@ -753,3 +753,20 @@ export function setHeader(init: RequestInit, name: string, value: string) {
         (init.headers as Record<string, string>)[name] = value;
     }
 }
+
+// very simple retrier utility for allowing some retries, if we ended up using it more often we should consider making it more elaborate
+export const retrier = (
+    operation: any, { attempts = Infinity, delayTime = 100 }) => new Promise((resolve, reject) => {
```

Comment:
Please convert this to `function ...` so that you can make it generic ( `function retrier<T>(...)` )
The `operation` argument should be typed as `Promise<T>`, or `()=>Promise<T>`,  or something.
No `any`.